src/minibook/main.py

```
@@ -9,15 +9,36 @@
+class GenerationParams(NamedTuple):
+    """Parameters for minibook generation."""
+
+    output_format: str
+    title: str
+    link_tuples: list[tuple[str, str]]
+    subtitle: str | None
+    output: str
+    template: str | None
+
+
+def validate_url_format(url: str) -> tuple[bool, str | None]:  # noqa: PLR0911
```

Ruff (0.14.13)

Unused `noqa` directive (non-enabled: `PLR0911`)

Remove unused `noqa` directive

(RUF100)